assets/src/containers/ResourcesAccessed.js

```diff
@@ -213,21 +227,30 @@ function ResourcesAccessed (props) {
     }
   }
 
-  const ResourceAccessChartBuilder = (resourceData) => {
+  const resourceAccessChartBuilder = (resourceData, resourcesLimit) => {
     if (resourceTypeFilter.length === 0) {
       return (<AlertBanner>Please select a resource type to display data.</AlertBanner>)
     } else if (!resourceData || Object.keys(resourceData).length === 0) {
       return (<AlertBanner>Resource data for your selections is not available.</AlertBanner>)
     } else {
       return (
-        <ResourceAccessChart
-          data={resourceData}
-          weekRange={weekRange}
-          gradeSelection={resourceGradeFilter}
-          resourceType={resourceTypeFilter}
-          aspectRatio={0.3}
-          minHeight={350}
-        />
+        <>
+          <Typography style={{ textAlign: 'center', verticalAlign: 'middle' }} gutterBottom>
+            {
+              resourcesLimit === undefined
+                ? 'Displaying all available resources in this course'
+                : `Displaying top ${resourcesLimit} most accessed resources in this course`
```

Comment:
Ternary could be moved into the string to reduce duplication; "Displaying ... resources in this course" is repeated.

Reply:
Yeah interesting, looks like I could just use
`            {`Displaying ${resourcesLimit ?? 'all available'} resources in this course`}`